tests/dir.props

```diff
@@ -42,20 +42,29 @@
   <PropertyGroup>
     <ValidatePackageVersions>true</ValidatePackageVersions>
     <ProhibitFloatingDependencies>true</ProhibitFloatingDependencies>
+
+    <CoreFxExpectedPrerelease>rc3-24117-00</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>beta-24314-02</CoreClrExpectedPrerelease>
+
+    <CoreFxVersionsIdentityRegex>^(?i)((System\..*)|(Microsoft\.CSharp)|(Microsoft\.NETCore.*)|(Microsoft\.Win32\..*)|(Microsoft\.VisualBasic))(?&lt;!TestData)$</CoreFxVersionsIdentityRegex>
   </PropertyGroup>
 
   <ItemGroup>
-    <ValidationPattern Include="^(?i)((System\..%2A)|(Microsoft\.CSharp)|(Microsoft\.NETCore.%2A)|(Microsoft\.Win32\..%2A)|(Microsoft\.VisualBasic))(?&lt;!TestData)$">
-      <ExpectedPrerelease>rc3-24117-00</ExpectedPrerelease>
+    <ValidationPattern Include="CoreFxVersions">
+      <IdentityRegex>$(CoreFxVersionsIdentityRegex)</IdentityRegex>
+      <ExpectedPrerelease>$(CoreFxExpectedPrerelease)</ExpectedPrerelease>
     </ValidationPattern>
-    <ValidationPattern Include="^(?i)(xunit(\.assert|\.core|\.runner\.(utility|msbuild))?)$">
+    <ValidationPattern Include="XunitPackageVersions">
+      <IdentityRegex>^(?i)(xunit(\.assert|\.core|\.runner\.(utility|msbuild))?)$</IdentityRegex>
       <ExpectedVersion>$(XunitPackageVersion)</ExpectedVersion>
     </ValidationPattern>
     <!-- Add a dummy value so that the item isn't removed by msbuild. Without the | this item doesn't show up later. -->
-    <ValidationPattern Include="^(?i)(xunit\.console\.netcore|dummy value)$">
+    <ValidationPattern Include="DummyVersions">
+      <IdentityRegex>^(?i)(xunit\.console\.netcore|dummy value)$</IdentityRegex>
```

Comment:
Ah, right, the `(...|dummy value)` was a workaround for msbuild not accepting the item (my theory is that adding the `|` forces msbuild not to treat the value as a path to search for). That part can be removed. A name like `XunitConsoleVersion` would make more sense, now. :smile: